docs/d/server_certificate.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "oneview"
+page_title: "Oneview: server_certificate"
+sidebar_current: "docs-server_certificate"
+description: |-
+ Gets information about an existing server_certificate.
+---
+
+# oneview\_server\_certificate
+
+Use this data source to access the attributes of a server certificate.
+
+## Example Usage
+
+```hcl
+data "oneview_server_certificate" "test" {
+ name = "Test server certificate"
```

Comment:
It should be either remote IP or alias name, here its just name which is incorrect.

Reply:
Done